thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -202,6 +201,8 @@ public Builder setInstallationID(
          *
          * @param app Kii Cloud Application.
          * @param owner Specify who uses the ThingIFAPI.
+         * @param actionTypes the format of key of actionTypes is "[alias]:[actionName]".
```

Comment:
@shinsuke-nara I improved the description in commit 34f3940 to make it clear. Can you check it?